Makefile

```diff
@@ -7,14 +7,18 @@
 #   - make check_release
 #   - make release
 
-.PHONY: clean sterile coverage docs help \
-	quality requirements test test-all upgrade validate
-
 .DEFAULT_GOAL := help
 
 # For opening files in a browser. Use like: $(BROWSER)relative/path/to/file.html
 BROWSER := python -m webbrowser file://$(CURDIR)/
 
+# This runs a Python command for every make invocation, but it's fast enough.
+# Is there a way to do it only when needed?
+VERSION := $(shell python -c "import build.util as bu; print(bu.project_wheel_metadata('.')['Version'])")
```

Comment:
https://pypi.org/p/pyproject-hooks might make more sense here.

Reply:
I don't know how I would use that.

Reply:
Yeah, that'd be more involved. You won't want to inline it.